alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/nodedao/SolrNodeDAOMetrics.java

```diff
@@ -0,0 +1,54 @@
+package eu.xenit.alfred.telemetry.binder.solr.nodedao;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import java.util.function.ToDoubleFunction;
+
+import org.alfresco.repo.domain.node.NodeDAO;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.transaction.TransactionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SolrNodeDAOMetrics {
```

Comment:
1. drop the `Solr`-part, it's not related to solr directly
2. find a better name for the remaining part :-)